src/main/java/com/server/RequestHeaderParser.java

```diff
@@ -44,6 +50,30 @@ private String extractPath() {
         return route.trim();
     }
 
+    private String extractBody(){
```

Comment:
Awesome! Now that we're also extracting the request body, what do you think about renaming this class to `RequestParser`? @ranizilpelwar & @shibani

Reply:
renamed